src/main/java/com/softserveinc/dokazovi/config/CorsConfig.java

```diff
@@ -9,10 +10,13 @@
 
     private static final long MAX_AGE_SEC = 3600;
 
+    @Value("${endpoints.cors.allowed-origins}")
+    private String [] allowedOrigins;
```

Comment:
Did you test it? Your build is failed

Reply:
Yes, everything was ok

Reply:
https://github.com/ita-social-projects/dokazovi-be/pull/191/checks?check_run_id=2023305740